test/controllers/comment_controller_test.exs

```diff
@@ -64,6 +64,28 @@ defmodule CodeCorps.CommentControllerTest do
     test "does not create resource and renders 403 when not authorized", %{conn: conn} do
       assert conn |> request_create(@valid_attrs) |> json_response(403)
     end
+
+    @tag :authenticated
+    test "tracks comment created", %{conn: conn, current_user: current_user} do
+      task = insert(:task)
+      attrs = @valid_attrs |> Map.merge(%{task: task, user: current_user})
+      comment_id = conn
+                    |> request_create(attrs)
+                    |> json_response(201)
+                    |> get_in(["data", "id"])
+                    |> String.to_integer
+
+      id = current_user.id
+      properties = %{
+        comment_id: comment_id,
+        task: task.title,
+        task_id: task.id,
+        task_type: task.task_type,
+        project_id: task.project_id
+      }
+
+      assert_received {:track, ^id, "Created Comment", ^properties}
```

Comment:
Ah, okay! No worries then. I knew when I was asking this that this might be the answer. ;)